### DISCUSSION TITLE
Support Delta Sharing on Delta Lake Trino Connector

### DISCUSSION THREAD

**Discussion by kuripart** [trinodb/trino#19153] [General]
Hi,

Opening this to discuss the support for: https://github.com/delta-io/delta-sharing in Trino.

1) Is this a feature that could be added to the existing Delta Lake Trino connector?
2) Is there any ongoing discussion, work, issue or ticket that is being used to track this?
3) Based on: https://trinodb.slack.com/archives/CP1MUNEUX/p1658943739298699 have there been other discussions to set this up as a different connector?

**Comment by ebyhr** [comment]
Closing as duplicate of #19179. Please avoid cross-posting.
